vendor/bat-native-ledger/src/bat/ledger/internal/database/database_server_publisher_info.cc

```diff
@@ -145,12 +145,6 @@ void DatabaseServerPublisherInfo::OnGetRecord(
   info->updated_at = GetInt64Column(record, 2);
   info->banner = banner.Clone();
 
-  // The `CONNECTED` status is deprecated. If this value appears in the database
-  // convert it to a valid status.
-  if (info->status == mojom::PublisherStatus::CONNECTED) {
-    info->status = mojom::PublisherStatus::NOT_VERIFIED;
```

Comment:
I think we should remove `CONNECTED` from `PublisherStatus` in `ledger.mojom`, too.

Reply:
Agreed. Since this will be uplifted, I would like to keep the number of files modified under control though. I've created an issue at https://github.com/brave/brave-browser/issues/25730 for removing the `CONNECTED` value.

Reply:
Alrighty. In this issue, we could also remove `CONNECTED` from `BraveRewardsPublisher` in `BraveRewardsPublisher.java`, and any iOS references, too.